sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/descriptor/MetricDescriptor.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.descriptor;
```

Comment:
this class is only used by the classes in the `internal.state` package. Does it need its own package?

Reply:
Not when viewed from this PR no.   Eventually this will be used/returned by `View`s.   

Specifically:  Right now the pre-existing `View` code does not allow metric renaming.  The new SDK specification does.  

The code you see that creates this today is a stop-gap for that implementation.